client/me/purchases/cancel-purchase/index.jsx

```diff
@@ -63,12 +64,53 @@ const CancelPurchase = React.createClass( {
 		page.redirect( redirectPath );
 	},
 
+	renderFooterText() {
+		const purchase = getPurchase( this.props ),
+			{ priceText, renewDate } = purchase,
+			renewalDate = this.moment( renewDate ).format( 'LL' );
```

Comment:
Probably not a big deal in term of performance but we could declare this date after the first `if` clause - so it's not instanciated nor formatted for nothing when the subscription is refundable.